Data/Sys/Themes/Boomy/README.txt

```diff
@@ -1 +0,0 @@
-Boomy: Milosz Wlazlo [miloszwl@miloszwl.com]
```

Comment:
Error. MaJoR didn't specify which READMEs to remove so I removed all of them. Fixing that in my next push.